server/prisma/prisma.yml

```diff
@@ -0,0 +1,20 @@
+# Specifies the HTTP endpoint of your Prisma API.
+#endpoint: ''
+#endpoint:
+endpoint: https://eu1.prisma.sh/anton-lunov/Buddy-app/test
```

Comment:
Would it be possible to create a new account that we can share?

Reply:
We can share this account, why not? I create it specially to use for Buddy-app. For sure, you can create your own if you wish and fill it with the data you want.

Reply:
It should be an ENV var.

Reply:
`Node env`